packages/taquito-rpc/src/taquito-rpc.ts

```diff
@@ -1172,4 +1173,25 @@ export class RpcClient implements RpcClientInterface {
       method: 'GET',
     });
   }
+
+  /**
+   *
+   * @param params contains the kind (PVM) and kernel to generate the origination proof from
+   * @description rpc call to generate the origination proof needed for the smart rollup originate operation
```

Comment:
nitpick,

for a* smart rollup originate op

I think the verbage 'the' can be misconstrued by users